packages/components/src/image/src/AsyncImage.tsx

```diff
@@ -38,14 +38,17 @@ function InnerAsyncImage({
         throw new Error("An async image retry count must be equal or greater to 1.");
     }
 
-    useCancellableEffect(isCancelled => {
+    useEffect(() => {
         const timeoutId = setTimeout(() => {
-            if(!isCancelled()) {
-                setCanRender(true);
-            }
+            setCanRender(true);
         }, delay);
 
         setCanRenderTimeoutId(timeoutId);
+
+        return () =>{
```

Comment:
Missing space before the {

Still doesn't understand why eslint let these pass! ;)